static/css/components/edit-toolbar--tablet.less

```diff
@@ -4,16 +4,21 @@
  */
 
 /* stylelint-disable selector-max-specificity */
+div.editionAll {
+  padding-bottom: 1rem;
+  display: flex;
+  justify-content: end;
+}
+
 div#editTools,
 div.editButton,
 div#editHistory {
+  display: flex;
```

Comment:
```suggestion
```
Remove `display: flex` and add `float: right` beneath the padding (revert to original styling).